poetry.lock

```diff
@@ -88,6 +88,17 @@ version = "1.4"
 [package.dependencies]
 pyflakes = ">=1.1.0"
 
+[[package]]
```

Comment:
Seem contains a lot of changes. Did you add the `invenio-record-resources` with `poetry add`?

Reply:
yes, I did.